handled the empty field or email/password

liamLacuna opened the pull request.

liamLacuna committed:
handled the empty field or email/password

liamLacuna committed:
handled the empty field or email/password

evanugarte commented:
Nice, but could you only show the emply email field and text only after the user tries to submit?

evanugarte commented on src/components/Auth/LoginScreen.js:
```diff
@@ -67,8 +67,8 @@ class LoginScreen extends React.Component {
           {authError ? "Login Failed." : ""}
-          {this.state.email}
-          {this.state.password}
+          {this.state.email === "" ? "Enter a Email.": ""}
```
have the text render only if a login was attempted with an empty field.

evanugarte reviewed (changes requested):
see comment above from friday

liamLacuna committed:
handled empty field

liamLacuna committed:
handled empty field

rickywraith committed:
Scroll view for expense list and manual add items (#32)

* Creates Add button

* Edits package.json to allow linking

* Started Revisions for Travis

Tried modifying package-lock to see if build fixes.

* Fixes lint errors

* Fixes missing lint package

* Adds scroll and other UI improvements

* Re-add package lock

* .

* Remove icon usage

* Revert package.json

* Node modules revert

evanugarte committed:
Travis Script Fix (#34)

* Attempted Travis Script Fix

Had travis update npm and clear any cache.

* Changed Node Version

evanugarte committed:
Added Receipt Camera Parsing (#35)

* Started Cloud Vision Text Read

* Added Better Cropping to Image

Now we can crop images to whatever size we want. Parsing text is next.

* Added Ability to Grab Text Response

Now we parse the response to only grab the text that Google saw. On to the next step.

* Added Total Extraction

We now can extract a total price from the OCR scan. Next is moving the
total to FormFields to add stuff to it. Yay

* Added No Value Handling To Camera

If the uri is undefined, we leave and do nothing.

* Added Cloud Vision File

Now you can never see it

* Added Proper Undefined Value Handling

Now we will only display real items in the expense list and alert the
user if we couldn't get anything from a scan.

* Finished Receipt Parsing

We now can take a photo of a receipt and from there get the values. DID
THAT!!!!!!!!!!!!!!!!

* Fixed SetState Bug

Wierd loop inariant, now its ok. Were lamost done

* removed console.log

evanugarte committed:
Added File Page + Form Handling (#37)

* Added Page to View Total Expense Price + Expenses

Now the user can see all of the expenses and screenshot them, exporting
them to a PNG.

* Style and File Export Structure

* Added Form Validation

Now we won't add any bad values to an item. Next is date

evanugarte committed:
Added Navigation + Fixed Form Bugs (#38)

* Added Navigation + Fixed Form Bugs

Bottom Tab navbar + empty item fields in form

* removed console

liamLacuna committed:
logout button moved in profile page

liamLacuna committed:
bugs

evanugarte commented on src/actions/expenseActions.js:
```diff
@@ -1,5 +1,13 @@
 //Import constant action types
-import { GET_EXPENSES, ADD_EXPENSE, DELETE_EXPENSE, EXPENSES_LOADING, EDIT_EXPENSE, SEARCH_EXPENSES } from "./types";
+import { 
```
these changes have already been added to master.

evanugarte commented on .travis.yml:
```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
- - "stable"
+ - "11"
```
these changes have already been added to master.

evanugarte reviewed (changes requested):
incorrect rebasing
